app/presenters/document_export_presenter.rb

```diff
@@ -108,6 +108,7 @@ def present_images(edition)
     edition.images.map do |image|
       image["alt_text"].squish! if image["alt_text"]
       image["caption"].strip! if image["caption"]
+      image["caption"] = nil if image["caption"] && image["caption"].empty?
```

Comment:
Might want to handle both of these things on the same line: e.g. `image["caption"] = image["caption"].presence&.strip`

Do we need to be doing the same thing with alt_text too? 

On the other hand should Content Publisher just set caption and alt_text to empty texts? I think that's how they end up in Content Publisher through the UI if someone doesn't fill them in.

Reply:
I'll update this to do both on the same line, since that's cleaner.

It's not an issue with alt_text:
https://github.com/alphagov/whitehall/blob/9396ec5a69750bd41c106fa7517d8c3476988d31/app/presenters/lead_image_presenter_helper.rb#L18-L24

I think it would be nice if Content Publisher could have these set as empty strings, however it would cause issues with the integrity check and would involve including an exception.